### ISSUE TITLE
Nothing happens when I select 'pick color from webpage'

### ISSUE THREAD

**Issue by NeonBeacon** [kepi/chromeEyeDropper#255]
### Describe the bug

When I select 'pick color from webpage', it looks to be loading something, but nothing happens and I cannot select any colors.

### Expected Behavior

It should show crosshairs cursor when the buitton is selected, however all it does is show a loading sign next to the arrow cursor but nothing happens past that.

### Steps To Reproduce

open the extension, select 'pick color from webpage' and hover over the colour you wish to select. Unsure why this does not work for me, I turned all other extensions off in order to see if there was a confliction, but still did not work.

### Screenshots

https://github.com/kepi/chromeEyeDropper/assets/129503880/c3ce355b-6e9d-4088-886e-97ee329a3fc7

### Operating system

Windows

### Browser version

latest

**Comment by kepi** [comment]
@NeonBeacon thanks for detailed description, this really helps.

I supposed you tried this, but can you please confirm, that you tried to restart the browser? You shouldn't need too, but I just want to be check if problem is in script initialization or elsewhere.

I see from your profile, that you are web developer. If you can work with dev tools, can you please try to check console on the page where you are trying to pick the color and if there is nothing, than try to go to `chrome://extensions/` page, turn on developer mode in top right corner and then click on **background page** link in box with *Eye Dropper* and check the console there. There might be nothing, but if there are any errors, it will really help to chase this down.

Btw. I'm just finishing big rewrite of Eye Dropper, it should be hopefully published in a few days. If you are, by any chance, interested in alpha/beta testing, let me know.

**Comment by NeonBeacon** [comment]
Hi Kepi,

Absolutely, I'm very fresh of the boot(camp), recently changed careers, so could do with all the experience I can get.
Let me know the details and I'll be happy to help out with testing. 

Regarding this current issue, yes, I tried rebooting chrome, still nothing, in the console, it shows the below error each time.

(webpage name)':1 Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.'

Seems it just doesn't want to connect for some reason, I've switched VPN off to see if that was the issue, but no dice either.

**Comment by kepi** [comment]
Cool, I'll have to setup some testing group. I'll let you know when I get to it.

Thanks for the error, I hit this while testing the rewritten version once, but wasn't able to reproduce it so far. I'll have to dig deeper. It seems to me like error in Chrome, but hopefully there would be some workaround possible.

**Comment by NeonBeacon** [comment]
No problem, look forward to hearing from you Kepi!

Have a good week!

**Comment by kepi** [comment]
Primarily for myself: I think this is a root cause of the problem: https://stackoverflow.com/q/54181734/259187

As changing this code in current version would be dangerous without more testing, I decided against it. This would have to wait for the rewrited version. I made substantial progress today, have all current features reimplemented. Now I just have to polish the interface and test more before releasing.

I'll check if it will be possible to publish alpha version for testing. Last time i tried it, it was rejected by Chrome Store :/

**Comment by kepi** [comment]
@NeonBeacon folks at Chrome Web Store unfortunatelly takes ages to verify the beta version. If you have time, it would help me if you can test extension from zip file I just published in releases: https://github.com/kepi/chromeEyeDropper/releases/tag/v1.2.0.0

Simply unzip it somewhere, go to `chrome://extensions`, click on **Developer mode** top right and then **Load unpacked** almost top left (below Extension heading).

I'm mostly interested if picking from web page works now, but as this is almost complete rewrite of ED which I would like to publish soon, any feedback is definitely welcomed, even small issues. Feel free to spam here or create new issues if you have time for it :)

Thanks

**Comment by NeonBeacon** [comment]
Hi Kepi,

The functionality works well, although my initial thought was that it
hadn't worked. I'll walk you through my thought process as a user
downloading the extension for the first time.

   - This time when selecting 'web page' the mouse icon changed and you
   could see a small box indicating the colour the mouse is hovering over
   (perfect, smiley face)
   - (This is where the process confused me) I had clicked on the colour,
   the icon disappeared, my first idea was to ctrl + v  to paste the hex code
   or rgb number as it may have copied it from the colour. I only did this as
   I'm used to that functionality in blender or adobe, unsure if other users
   would also assume this.
   - I then tried the same process again, not realising it had saved the
   colour in the 'selected' box once I selected colour picker again. (This is
   likely my fault for not reading how to use the extension, but potentially
   food for thought regardless).
   - 3rd time round I realised where the colour stores and shows the codes
   for the colours and where to copy them for usage.
   - As the extension box disappears after you select 'from webpage' (which
   is understandable so to be able to select anything on the webpage) you
   don't see the code saved until you click back into the extension, which can
   be confusing using the extension for the first time.


Haha now having written all the above, I've now explored the options and
the rest of the well thought out functionality of your extension, and 'copy
colour to clipboard' is indeed an option. Sorry to waste your time.

I guess my only potential critique for functionality would be to have that
set as ticked as standard when someone installs the extension. But again,
that might just be my preference rather than anything else.

The only other area I would mention, with a design background, only
cosmetic, regarding the extension drop down frame,slightly rounded edges to
the drop down box would be less harsh, and the font for the 'hex', 'code',
'rgb', 'hsl', 'hsv', 'New', 'Selected', 'Pick from', 'Hover over colors to
preview a color in New square' and the 'linear-gradient' text could be
prettier. But again, the cosmetics for such a good tool aren't that
important.

I've tested a few other controls on the extension and all seems to work
great!

Thanks for allowing me to test. Unsure if you preferred this all on Github
or on here?

Let me know if you want any other testing done, more than happy to help
out.

Thanks

Dave
